release/models/mpls/openconfig-mpls-ldp.yang

```diff
@@ -918,6 +942,32 @@ module openconfig-mpls-ldp {
     }
   }
 
+  grouping mpls-ldp-label-advertisement-config {
+    description
+      "Grouping containing configuration to enable Downstream on demand in a session.";
+    leaf downstream-on-deman-enabled {
```

Comment:
```suggestion
    leaf downstream-on-demand-enabled {
```

Discussion point -- is downstream-on-demand common enough that we might be able to just call this `dod-enabled`?

Reply:
For the discussion point.... I am happy with both :-)

Reply:
One point I am not sure is if the name (or the leaf itself) is clear for the state. In the state container,  we could have instead of the downstream-on-demand-enabled leaf, have a local-label-advertisement mode.   downstream-on-demand-enabled is only the local preference, but as explained, the negotiated label adverstisement mode is the one used...

Reply:
ACK - one option could be:

 * `downstream-on-demand-requested` or `dod-requested` - the `config` leaf that specifies whether DOD is to be used.
 * `negotiated-advertisement-mode` as the `state`-only leaf that covers what the resulting negotiation is.